README.md

```diff
@@ -7,7 +7,7 @@
 
 Bridge-In-Tech is an application that allows industries/companies, mentors and students to actively collaborate to one another. 
 
-This is the backend client of Bridge-In-Tech.
+This is the backend client of [Bridge-In-Tech Web](https://github.com/anitab-org/bridge-in-tech-web).
```

Comment:
https://github.com/anitab-org/bridge-in-tech-web/blame/5ea9a879c9b6952be3fd918727a7ca5b6d533bdd/README.md#L8

After reading the above sentence I think this line needs to be restructured.
What do you think @epicadk. @devkapilbansal and @decon-harsh?

```suggestion
This is the backend client of [Bridge-In-Tech-Web](https://github.com/anitab-org/bridge-in-tech-web).
```
Also, it will be good if the slug will be continuous.

Reply:
@codesankalp Kindly review it again

Reply:
instead of `client` we can write "repository" sounds more suitable

Reply:
@Amulya-coder in my opinion, client is fine here